decidim-module-census_connector/spec/system/verifications/census_registration_spec.rb

```diff
@@ -10,7 +10,7 @@
 
   let!(:inner_scope) { create(:scope, code: Decidim::CensusConnector.census_local_code, organization: organization) }
   let!(:exterior_scope) { create(:scope, code: "XX", organization: organization) }
```

Comment:
I guess we should use the configuration option here for consistency with the previous line.

Reply:
Yep, I saw that line, but didn't see the previous line :see_no_evil: